src/main.ts

```diff
@@ -44,7 +44,9 @@ async function run(): Promise<void> {
   }
 }
 
-function setOutputs(isFound: boolean, source: ESource | null): void {
+function setOutputs(key: string, source: ESource | null): void {
+  var isFound = key !== '';
+  core.setOutput('jira-issue-key', key);
```

Comment:
I suggest to pass `null` instead of an empty string here, it's more straightforward IMHO

Reply:
> Argument of type 'null' is not assignable to parameter of type 'string'.ts(2345)
I originally had it that way but my text editor is complaining (I'm not a javascript developer).

Reply:
it's a typescript error :)

please use `key: string | null` and it'd work well